synfig-studio/plugins/lottie-exporter/settings.py

```diff
@@ -71,12 +72,13 @@
 UNKNOWN_LAYER = "unknown_layer"
 CONVERT_METHODS = {"add", "atan2","average", "bone", "bone_link", "bone_root", "composite", "cos", "exp", "fromint", "linear", "radial_composite", "scale", "sine", "subtract", "switch", "vectorangle","weighted_average"}
 BONES = {"bone", "bone_root"}
-
+BLUR_TYPE = 29
 # Some waypoint animated definitions
 ANIMATED = 2
 SINGLE_WAYPOINT = 1
 NOT_ANIMATED = 0
-
+GROUP_FLAG = False #indicates whether layer is in a group or not
```

Comment:
We already had a `INSIDE_PRECOMP` flag. Can you explain we need only group layer flag specifically?

Reply:
I noticed `INSIDE_PRECOMP` now, my bad. I removed `GROUP_FLAG` and used `INSIDE_PRECOMP` and it does the job for me.

Reply:
Great!